python/helper-image/launcher/launcher.go

```diff
@@ -126,11 +146,16 @@ func main() {
 		logrus.Fatal("expected python command-line args")
 	}
 	pc.args = flag.Args()
-	logrus.Tracef("command-line: %v", pc.args)
+	logrus.Debug("app command-line: ", pc.args)
 
-	if err := pc.launch(ctx); err != nil {
-		logrus.Fatalf("error launching python debugging: %v", err)
+	if !pc.prepare(ctx) {
```

Comment:
Out of curiosity if prepare fails, why does it make sense to launch the original command vs being a fatal error?

Reply:
If we're unable to configure the app for debugging, then we just launch the original command without debugging.  Otherwise for microservices-type apps, the failure to debugify a container can lead to the app failing.

#81 includes pointers to failure cases with Bank of Anthos and Hipster Shop.